### DISCUSSION TITLE
Allow mediation case to be closed by trading peer

### DISCUSSION THREAD

**Discussion by ghost** [bisq-network/bisq#6873] [Ideas]
Introduce the feature to allow the mediation case to be opened by the trading peer that has opened the case.

There are cases when the trading peer that opened the mediation case what to close it to close the trade. Currently it is not possible, a mediator is required to attend and resolve.

The suggested feature is for giving the initiator the possibility to close it and resume the trading from where it was before opening it.
This may reduce mediators workload and may reduce waiting time for trading peers.

**Comment by pazza83** [comment]
Would be good, but the UX would need to make sure that new users were not taken advantage of somehow
